features/managing_queue_items/browsing_queue_items.feature

```diff
@@ -1,44 +0,0 @@
-@managing_queue_items
-Feature: Browsing queue items
-  In order to see the status of imported and not imported items from Akeneo
-  As an Administrator
-  I want to browse the Akeneo items queue
```

Comment:
Non lo farei in questa PR ma a mio avviso ci vorrà qualcosa in futuro per avere un'interfaccia di monitoraggio dell'andamento dell'import da Akeneo.

Adesso con l'introduzione del messenger la perdiamo. Dovremo studiare qualcosa.